settings/api/routes/routes.go

```diff
@@ -0,0 +1,11 @@
+package routes
+
+import (
+	"api/controller"
+	"github.com/julienschmidt/httprouter"
+)
+
+func Register(rtr *httprouter.Router, ctrl *controller.Controller) {
+	rtr.GET("/view", ctrl.Viewer.View)
+	rtr.PUT("/update", ctrl.Updater.Update)
```

Comment:
No kind of authentication is there on this route

Reply:
This registers routes on the router by using handlers from controller. It dont think it nees authentication?